src/config/containers-and-vms/chroot.md

```diff
@@ -0,0 +1,90 @@
+# Creating and using chroots and containers
+
+chroots and containers can be set up and used for many purposes, including:
+
+- running glibc software on musl (and vice versa)
+- running software in a more controlled or sandboxed environment
+- creating a rootfs for bootstrapping a system
+
+## Chroot Creation
+
+Create a directory that will contain the chroot, then install a base system in
+it via the `base-voidstrap` package:
+
+```
+$ XBPS_TARGET_ARCH=<chroot_arch> xbps-install -M -r <chroot_dir> -R <repository> base-voidstrap
+```
+
+The `<repository>` varies depending on architecture:
+
+- `aarch64`, `aarch64-musl`:
+   `https://repo-default.voidlinux.org/current/aarch64`
+- any other `musl` architecture:
+   `https://repo-default.voidlinux.org/current/musl`
+- any other architecture: `https://repo-default.voidlinux.org/current`
+
+Alternatively, this can be done automatically with
+[`xvoidstrap(1)`](https://man.voidlinux.org/xvoidstrap.1) (from `xtools`):
+
+```
+# XBPS_ARCH=<chroot_arch> xvoidstrap <chroot_dir> base-voidstrap
+```
+
+## Chroot Usage
+
+### xchroot
+
+[`xchroot(1)`](https://man.voidlinux.org/xchroot.1) (from `xtools`) can be used
+to automatically set up and enter the chroot.
+
+### Manual Method
+
+Alternatively, this process can be done manually.
+
+If network access is required, copy `/etc/resolv.conf` into the chroot;
+`/etc/hosts` may need to be copied as well.
+
+Several directories then need to be mounted as follows:
+
+```
+# mount -t proc none <chroot_dir>/proc
+# mount -t sysfs none <chroot_dir>/sys
+# mount --rbind /dev <chroot_dir>/dev
+# mount --rbind /run <chroot_dir>/run
+```
+
+Use [chroot(1)](https://man.voidlinux.org/chroot.1) to change to the new root,
+then run programs and do tasks as usual. Once you've finished using it, unmount
+the chroot using [umount(8)](https://man.voidlinux.org/umount.8).
+
+### Alternatives
+
+#### Bubblewrap
+
+[bwrap(1)](https://man.voidlinux.org/bwrap.1) (from the `bubblewrap` package)
+has additional features like the ability for sandboxing and does not require
+root access.
+
+`bwrap` is very flexible and can be used in many ways, for example:
+
+```
+$ bwrap --bind <chroot_dir> / \
+	--dev /dev \
+	--proc /proc \
+	--bind /sys /sys \
+	--bind /run /run \
+	--ro-bind /etc/resolv.conf /etc/resolv.conf \
+	<command>
+```
+
+The `bwrap` manpage and the [Arch Wiki
+article](https://wiki.archlinux.org/title/Bubblewrap#Usage_examples) contain
+more examples of `bwrap` usage.
+
+#### Application Containers
+
+If a more integrated and polished solution is desired, Void also [provides OCI
+containers](https://github.com/void-linux/void-docker/pkgs/container/void-linux)
+that work with tools like [docker](https://www.docker.com) and
+[podman](https://man.voidlinux.org/podman.1). These containers do not require
```

Comment:
docker links to docker's website, but podman links to podman manpage, I think this should be more consistent if possible

Reply:
there is no docker manpage

Reply:
It looks like an issue in man.voidlinux.org, I have it on my system and it's in repositories.
```
> xlocate man1/docker.1
docker-cli-20.10.22_1	/usr/share/man/man1/docker.1
```

Reply:
seems to be some kind of issue in xmandump. I think leaving it as-is for now is the best option